proton

```diff
@@ -0,0 +1,22 @@
+#!/bin/sh
+
+# List of python commands to try, in order
+pythons="python3 python python2"
+
+# Get proton's path, relative to this script's
+# Note: this won't work if it was called from the $PATH
+PROTON_PATH=$(dirname "$0")/proton.py
+
+for python in $pythons
+do
+	# Try every python. The "x" is here to avoid an empty string
+	if [ $($python -c "print('success')" 2>/dev/null )x = "success"x ]
```

Comment:
why not something like `if [ -x "$(command -v $pybin)" ]` ? The current solution works but seems hacky

Reply:
I thought about this as well, but this has the nice side effect of ensuring that `$python` really works as intended, at a marginal speed tradeoff. This doesn't really feel hacky to me, but I'm open to discussion.